src/components/BannerImage/BannerImage.tsx

```diff
@@ -1,11 +1,42 @@
-import { BannerContainer, Banner } from "./BannerImage.styles";
+import { IMainBanner } from "@/utils/interfaces";
+import { Icon } from "../Icon/Icon";
+import Image from "next/image";
+import { LinkButton } from "../LinkButton/LinkButton";
+
+const MainBanner = ({ content }: { content: { fields: IMainBanner } }) => {
+  console.log(content);
+  const { title, subtitle, image } = content.fields;
 
-const BannerImage = () => {
   return (
-    <BannerContainer>
-      <Banner src={"/banner.png"} />
-    </BannerContainer>
+    <div className="overflow-hidden relative flex justify-center items-center w-full mt-4 min-h-[450px]">
+      <Image
+        className="absolute w-full min-h-[450px] h-full object-cover rounded-t-md md:rounded-l-md md:rounded-tr-none z-0"
+        src={`https:${image.fields.file.url}`}
+        alt=""
+        width={1400}
+        height={1400}
+      />
+      <div className="absolute top-0 right-0 h-full w-[100%] bg-gradient-to-l from-black via-transparent to-transparent"></div>
+      <div className="absolute top-0 left-0 h-full w-[140%] bg-gradient-to-r from-black via-transparent to-transparent"></div>
+      <div className="flex w-full p-6 2xl:p-0 h-full items-end justify-between max-w-[1440px] z-10">
+        <div className="flex flex-col gap-4">
```

Comment:
```suggestion
        <div className="flex flex-col gap-4 lg:gap-8">
```
In Figma, this gap in desktop is 32px (gap-8)